actions/me/scripts.tsx

```diff
@@ -114,16 +113,3 @@ export async function getScriptContent(scriptURL: string): Promise<string | unde
         return undefined;
     }
 }
-
-export async function getNewScriptName() {
```

Comment:
The logic of magically delecting assistant name is removed. Right now it defaults to `New Assistant` where user can have multiple.